### ISSUE TITLE
Provide automated method for "cleaning" a node

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#373]
Originally reported of Google Code with ID 317.

```
Before the node-add changes for 2.7, one could force the addition of a node, even if it had old files lying around. There should be a tool to do this cleanup now that 2.7 no longer allows forcing the addition.
```
Originally added on 2012-11-21 09:26:46 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit d5fe5fa
Author: Michael Hanselmann <hansmi@google.com>
Date:   Wed Dec 5 16:02:11 2012 +0100

    Add tool to clean up node
    
    Sometimes a node is not removed properly from a cluster (especially
    during development). This new tool stops all daemons and removes (after
    making copies) the most critical files.
    
    Signed-off-by: Michael Hanselmann <hansmi@google.com>
    Reviewed-by: Iustin Pop <iustin@google.com>

```
Originally added on 2012-12-07 12:52:13 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
